science/projects/soroush_heidary/01-main-content/Main-Content.ipynb

```diff
@@ -0,0 +1,419 @@
+{
```

Comment:
<p><strong>Great work! Keep it up.</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Zarebin/nextgen-06/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>